components/calendly_v2/actions/create-scheduling-link/create-scheduling-link.mjs

```diff
@@ -0,0 +1,37 @@
+/* eslint camelcase: 0 */
+
+import calendly_v2 from "../../calendly_v2.app.mjs";
+
+export default {
+  key: "calendly_v2-create-scheduling-link",
+  name: "Create a Scheduling Link",
```

Comment:
We can keep the name of the app prop `calendly`. It just needs to be `calendly_v2` in the `key`.
```suggestion
import calendly from "../../calendly_v2.app.mjs";

export default {
  key: "calendly_v2-create-scheduling-link",
  name: "Create a Scheduling Link",
```